.github/workflows/scan-pull-request.yml

```diff
@@ -0,0 +1,47 @@
+name: "Frogbot Scan Pull Request"
```

Comment:
Is this able to be expanded to non-python searching as well? It seems like it only works for scanning python depends. Is it able to search the file system for licenses? For non-python projects, this would be super handy (C++).

Reply:
It appears so: https://jfrog.com/help/r/jfrog-artifactory-documentation/scan-c/c-builds

However it looks like there might be a few extra steps involved there, probably worth its own PBI. I'll make one